DuggaSys/microservices/sharedMicroservices/retrieveUsername_ms.php

```diff
@@ -13,19 +13,18 @@
 pdoConnect();
```

Comment:
The microservice works without calling this function. This is because the PDO that is passed as a parameter. I think we can presume that the PDO is already previously initalized by the service calling this function.